agbenchmark/start_benchmark.py

```diff
@@ -135,21 +135,37 @@ def start(
         )
         return 1
 
-    if not os.path.exists(CONFIG_PATH) or os.stat(CONFIG_PATH).st_size == 0:
+    if os.path.exists(CONFIG_PATH) and os.stat(CONFIG_PATH).st_size:
```

Comment:
I changed the work with the config little bit, I didn't want to overwrite the file, if you are missing just the new attributes